### PR TITLE
Drop unnecessary Linux config items

### PR DESCRIPTION
**Pull request by silverjam**
This slims down the image size by about 645 KBytes

### CODE REVIEW

**Comment by anonymous-travis-swiftnav** [comment]
## v1.3.0-develop-2018020221-2-gc6c36d3
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.3.0-develop-2018020221-2-gc6c36d3/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.3.0-develop-2018020221-2-gc6c36d3/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.3.0-develop-2018020221-2-gc6c36d3

**Comment by mfine** [comment]
Sweet

File: board/piksiv3/linux.config [hunk 1]
```diff
@@ -150,33 +149,10 @@ CONFIG_WATCHDOG=y
...
-CONFIG_VIDEO_XILINX_SWITCH=y
-CONFIG_VIDEO_XILINX_TPG=y
-# CONFIG_MEDIA_SUBDRV_AUTOSELECT is not set
-CONFIG_VIDEO_ADV7604=y
-# CONFIG_DVB_TUNER_DIB0070 is not set
-# CONFIG_DVB_TUNER_DIB0090 is not set
-CONFIG_DRM=y
-CONFIG_DRM_XILINX=y
+# CONFIG_VGA_ARB is not set
+CONFIG_FB=y
```
Comment (gsmcmullin) [inline board/piksiv3/linux.config]: We can probably remove this framebuffer stuff too.

**Comment by anonymous-travis-swiftnav** [comment]
## v1.3.0-develop-2018020221-5-g2579eba
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.3.0-develop-2018020221-5-g2579eba/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.3.0-develop-2018020221-5-g2579eba/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.3.0-develop-2018020221-5-g2579eba

**Comment by anonymous-travis-swiftnav** [comment]
## v1.3.0-develop-2018020221-6-ge315100
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.3.0-develop-2018020221-6-ge315100/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.3.0-develop-2018020221-6-ge315100/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.3.0-develop-2018020221-6-ge315100

**Comment by gsmcmullin** [comment]
lgtm

**Pull request merged**